working/augmentation-libraries/feature-specification.md

```diff
@@ -107,7 +107,7 @@ declaration *B* if and only if:
 
 *   *A* and *B* occur in the same file, and the start of *A* is syntactically
     before the start of *B*, in source order, or
-*   The file containing *A* includes the file that contains *B*.
+*   The file where *A* occurs includes the file where *B* occurs.
```

Comment:
Another tweak. ;-)

I don't think there's any support for claiming that 'includes' is transitive. 'Includes F' basically means that there is a directive of the form `part of 'F';`. So the safe bet would be something like this:

```suggestion
*   The file where *A* occurs includes a file that contains *B*.
```

This means that *B* could occur at any level which is strictly deeper than *A* in the file tree, not just one level deeper.